src/Windowing/Silk.NET.GLFW/GlfwNativeWindow.cs

```diff
@@ -81,5 +89,6 @@ public unsafe GlfwNativeWindow(Glfw api, WindowHandle* window) : this()
         public nint? Glfw { get; }
         public nint? Sdl { get; }
         public nint? DXHandle { get; }
+        public (nint? display, nint? surface)? EGL { get; }
```

Comment:
Can these be PascalCase 
```suggestion
        public (nint? Display, nint? Surface)? EGL { get; }
```
(everywhere)